pkg/agent/util/net_linux.go

```diff
@@ -237,3 +239,37 @@ func DeleteOVSPort(brName, portName string) error {
 	cmd := exec.Command("ovs-vsctl", "--if-exists", "del-port", brName, portName)
 	return cmd.Run()
 }
+
+func RenameInterface(from, to string) error {
+	klog.InfoS("Rename interface", "oldName", from, "newName", to)
```

Comment:
```suggestion
	klog.InfoS("Renaming interface", "oldName", from, "newName", to)
```
to indicate it's going to rename it, not done yet.

Reply:
Fixed. Thanks.